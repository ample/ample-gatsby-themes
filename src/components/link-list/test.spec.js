import React from "react"
import renderer from "react-test-renderer"

import LinkList from "."
import Fixture from "./_fixtures"

describe("LinkList", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<LinkList heading={Fixture.vertical.label} links={Fixture.vertical.items} vertical />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
