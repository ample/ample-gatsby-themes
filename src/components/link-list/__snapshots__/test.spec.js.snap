// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkList renders correctly 1`] = `
<ul
  className="cl_link_list cl_link_list_is_vertical"
>
  <li>
    <strong>
      Quisque
    </strong>
  </li>
  <li>
    <a
      href="/"
    >
      Vestibulum vitae
    </a>
  </li>
  <li>
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      Pellentesque tempor venenatis
    </a>
  </li>
  <li>
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      Fusce molestie
    </a>
  </li>
  <li>
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      Etiam in lacinia dolor
    </a>
  </li>
</ul>
`;
